src/NHibernate/Loader/Loader.cs

```diff
@@ -646,9 +646,11 @@ private IEnumerable<SubselectFetch> CreateSubselects(List<EntityKey[]> keys, Que
 			}
 		}
 
-		internal void InitializeEntitiesAndCollections(
-			IList hydratedObjects, DbDataReader reader, ISessionImplementor session, bool readOnly,
-			CacheBatcher cacheBatcher)
+		public void InitializeEntitiesAndCollections(IList hydratedObjects, 
+		                                             DbDataReader reader, 
+		                                             ISessionImplementor session, 
+		                                             bool readOnly,
+		                                             CacheBatcher cacheBatcher)
```

Comment:
The previous formatting has a reason: with tab indentation, it is a lot more stable than this new formatting, which tends to "break" when some reformatting tool change spaces back to tabs. Please keep previous formatting, or change it with first parameter on its own line:

```suggestion
		public void InitializeEntitiesAndCollections(
			IList hydratedObjects, 
			DbDataReader reader, 
			ISessionImplementor session, 
			bool readOnly,
			CacheBatcher cacheBatcher)
```